2023-05-19 10:20:06,812 INFO    MainThread:104757 [wandb_setup.py:_flush():76] Current SDK version is 0.15.3
2023-05-19 10:20:06,813 INFO    MainThread:104757 [wandb_setup.py:_flush():76] Configure stats pid to 104757
2023-05-19 10:20:06,813 INFO    MainThread:104757 [wandb_setup.py:_flush():76] Loading settings from /home/edo/.config/wandb/settings
2023-05-19 10:20:06,813 INFO    MainThread:104757 [wandb_setup.py:_flush():76] Loading settings from /home/edo/Desktop/orc/03_assignment/wandb/settings
2023-05-19 10:20:06,813 INFO    MainThread:104757 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'_executable': '/usr/bin/python3'}
2023-05-19 10:20:06,813 INFO    MainThread:104757 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'DQN template.py', 'program': 'DQN template.py'}
2023-05-19 10:20:06,813 INFO    MainThread:104757 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2023-05-19 10:20:06,813 INFO    MainThread:104757 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2023-05-19 10:20:06,813 INFO    MainThread:104757 [wandb_init.py:_log_setup():507] Logging user logs to /home/edo/Desktop/orc/03_assignment/wandb/run-20230519_102006-vpbrswf8/logs/debug.log
2023-05-19 10:20:06,813 INFO    MainThread:104757 [wandb_init.py:_log_setup():508] Logging internal logs to /home/edo/Desktop/orc/03_assignment/wandb/run-20230519_102006-vpbrswf8/logs/debug-internal.log
2023-05-19 10:20:06,813 INFO    MainThread:104757 [wandb_init.py:init():547] calling init triggers
2023-05-19 10:20:06,813 INFO    MainThread:104757 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-05-19 10:20:06,813 INFO    MainThread:104757 [wandb_init.py:init():596] starting backend
2023-05-19 10:20:06,813 INFO    MainThread:104757 [wandb_init.py:init():600] setting up manager
2023-05-19 10:20:06,818 INFO    MainThread:104757 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-05-19 10:20:06,943 INFO    MainThread:104757 [wandb_init.py:init():606] backend started and connected
2023-05-19 10:20:06,947 INFO    MainThread:104757 [wandb_init.py:init():700] updated telemetry
2023-05-19 10:20:06,965 INFO    MainThread:104757 [wandb_init.py:init():737] communicating run to backend with 60.0 second timeout
2023-05-19 10:20:07,545 INFO    MainThread:104757 [wandb_run.py:_on_init():2177] communicating current version
2023-05-19 10:20:07,717 INFO    MainThread:104757 [wandb_run.py:_on_init():2186] got version response 
2023-05-19 10:20:07,718 INFO    MainThread:104757 [wandb_init.py:init():787] starting run threads in backend
2023-05-19 10:20:07,900 INFO    MainThread:104757 [wandb_run.py:_console_start():2158] atexit reg
2023-05-19 10:20:07,901 INFO    MainThread:104757 [wandb_run.py:_redirect():2013] redirect: SettingsConsole.WRAP_RAW
2023-05-19 10:20:07,901 INFO    MainThread:104757 [wandb_run.py:_redirect():2078] Wrapping output streams.
2023-05-19 10:20:07,901 INFO    MainThread:104757 [wandb_run.py:_redirect():2103] Redirects installed.
2023-05-19 10:20:07,902 INFO    MainThread:104757 [wandb_init.py:init():829] run started, returning control to user process
2023-05-19 10:20:07,903 INFO    MainThread:104757 [wandb_watch.py:watch():51] Watching
2023-05-19 10:20:07,904 ERROR   MainThread:104757 [util.py:get_module():226] Error importing optional module torch
Traceback (most recent call last):
  File "/home/edo/.local/lib/python3.8/site-packages/wandb/util.py", line 188, in import_module_lazy
    return sys.modules[name]
KeyError: 'torch'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/edo/.local/lib/python3.8/site-packages/wandb/util.py", line 221, in get_module
    return import_module_lazy(name)
  File "/home/edo/.local/lib/python3.8/site-packages/wandb/util.py", line 192, in import_module_lazy
    raise ModuleNotFoundError
ModuleNotFoundError
